sdks/python/apache_beam/yaml/examples/testing/examples_test.py

```diff
@@ -782,9 +853,67 @@ def _db_io_read_test_processor(
   return test_spec
 
 
+@YamlExamplesTestSuite.register_test_preprocessor(
```

Comment:
Is it possible to add these as tests embedded in the YAML file under a "tests:" section instead of being implemented in Python ? 

@derrickaw might know more.